js/dominion.js

```diff
@@ -368,9 +370,51 @@
         return this;
       },
 
-    next: function (e) { },
-    prev: function (e) { },
-    select: function (e) { },
+    next: function (e)
+      {
+        var $items = this.$menu.find('li');
+        var $active = this.$menu.find('li.active').first();
+
+        if ($active.length)
+        {
+          $active.removeClass('active');
+          $($items.get($items.index($active) + 1)).addClass('active');
```

Comment:
`jQuery.get(index)` returns the DOM element but `jQuery.eq(index)` returns a jQuery object, so you don't have to call the constructor again.